# es-roikoren/no-math-log1p
> disallow the `Math.log1p` method.

- âœ… The following configurations enable this rule: `plugin:es-roikoren/no-new-in-es2015`, `plugin:es-roikoren/restrict-to-es3`, and `plugin:es-roikoren/restrict-to-es5`

This rule reports ES2015 `Math.log1p` method as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
/*eslint es-roikoren/no-math-log1p: error */
const n = Math.log1p(value);
```

## ðŸ“š References

- [Rule source](https://github.com/roikoren755/eslint-plugin-es/blob/v2.0.5/src/rules/no-math-log1p.ts)
- [Test source](https://github.com/roikoren755/eslint-plugin-es/blob/v2.0.5/tests/src/rules/no-math-log1p.ts)
