# es-roikoren/no-regexp-u-flag
> disallow RegExp `u` flag.

- âœ… The following configurations enable this rule: `plugin:es-roikoren/no-new-in-es2015`, `plugin:es-roikoren/restrict-to-es3`, and `plugin:es-roikoren/restrict-to-es5`

This rule reports ES2015 RegExp `u` flag as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

```js
/*eslint es-roikoren/no-regexp-u-flag: error */
const r1 = /[â˜€ï¸â˜”]/u;
```

## ğŸ“š References

- [Rule source](https://github.com/roikoren755/eslint-plugin-es/blob/v3.0.0/src/rules/no-regexp-u-flag.ts)
- [Test source](https://github.com/roikoren755/eslint-plugin-es/blob/v3.0.0/tests/src/rules/no-regexp-u-flag.ts)
